spring-cloud-gcp-data-firestore/src/main/java/com/google/cloud/firestore/PublicClassMapper.java

```diff
@@ -60,7 +60,8 @@ private PublicClassMapper() {
 			decodedValue = convertToDateIfNecessary(decodedValue);
 			decodedFields.put(entry.getKey(), decodedValue);
 		}
-		return CustomClassMapper.convertToCustomClass(decodedFields, clazz);
+		return CustomClassMapper.convertToCustomClass(decodedFields, clazz,
+				new DocumentReference(null, ResourcePath.create(doc.getName())));
```

Comment:
stray line from a different PR?

Reply:
I upgraded to the latest Google Cloud Java BOM, and this line is needed to fix a compilation error. You can argue that the version update should be in its own PR, but c'mon. :smile:

Reply:
Ooh, that makes sense. I thought because firestore PR was also in flight that you were experimenting.